### DISCUSSION TITLE
API reference index page to use for LLM docs indexing

### DISCUSSION THREAD

**Discussion by hyperfocus1337** [directus/directus#23727] [Feature Requests]
### Summary

I'm trying to find a link to use as custom docs in my Cursor editor, to allow it to index the entire Directus API reference.

But there's no index page, that links to all the underlying API reference pages, thus I'm not able to get Cursor to index all the API docs.

![image](https://github.com/user-attachments/assets/e45ea241-04ee-4b4d-a3d8-40144ebbb2a8)

### Motivation

Being able to get improved LLM answers by indexing the Directus docs for LLM use.

### Detailed Design

No detailed design.

### Requirements List

An index page for all docs or API references (if it doesn't currently exist, that I'm unaware of).

Like an RSS feed / sitemap including all links.

### Drawbacks

None that I'm aware of.

### Alternatives

None yet.

### Adoption Strategy

None yet.

**Comment by br41nslug** [comment]
There already exists a sitemap for the docs https://docs.directus.io/sitemap.xml

**Reply by hyperfocus1337** [reply]
Thanks, it seems to be busy indexing those pages now.

![image](https://github.com/user-attachments/assets/9ae7617b-870f-4ccf-972c-8e07a5bf2cb6)

And it seems to be quite busy, eating up a lot of CPU on my M1 Mac.

![image](https://github.com/user-attachments/assets/fefd891d-2828-4c22-99b9-86e15f2fcf91)

Curious to see how long it will take to complete, if ever... 😂

**Reply by br41nslug** [reply]
There are a lot of pages... but i wonder if the sitemap also includes the typedoc generated part of the docs 😂 that would add a giant chunk of pages
